src/components/CategoryPicker.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+export default function CategoryPicker(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="md">
+      <div className={classes.paper}>
+        <Typography component="h2" variant="h6">
+          Search with a Relationship
+        </Typography>
+        <Typography>
+          Hold shift while clicking to select more than one category to include{' '}
+        </Typography>
+        <form className={classes.form} noValidate>
+          <Grid container spacing={2}>
+            <Grid item xs={12}>
```

Comment:
I'm not very good at layout, but it seems strange to have a Grid inside a Grid with just one element. It may be the right thing to do... HTML/CSS layout can be very mysterious! But it seems weird to me.

Reply:
Sorry and yes! It is weird and I did change in my update! :)